### PR TITLE
Add Auth0

### PR DESCRIPTION
**Pull request by ericamcarvalho**
# Added Auth0

I am going to add a ticket for this tomorrow. I added signup/signin and signout journeys as well as a protected route. Not sure how to style the header. If I keep the logo in the middle, it gets messy on mobile - so I left it as is. It would probably be good to move the logo to the left.

### Screenshots

Screenshot 2022-05-17 at 19 31 17

Screenshot 2022-05-17 at 19 31 09

Screenshot 2022-05-17 at 19 31 27

## Effected Areas

Auth feature, Router, App.

## Developer Checklist

I have

- [x] Made the minimum amount of change required to achieve my goal to a high standard
- [ ] Added tests that describe the change
- [ ] Created tickets for any tech debt incurred

### CODE REVIEW

**Review by KhalidStonehouse2001 — APPROVED** [review]
Amazing✅

**Comment by ericamcarvalho** [comment]
I added a call to the protected api endpoint. I'll wait until that one is merged to test if everything works fine before merging this one.

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**
